api/diagnostics_test.go

```diff
@@ -140,7 +140,7 @@ func TestDispatchLogsForCommand(t *testing.T) {
 
 	data, err := ioutil.ReadAll(r)
 	require.NoError(t, err)
-	assert.Equal(t, "OK\n", string(data))
+	assert.Equal(t, "OK\r\n", string(data))
```

Comment:
Now it's failing here https://ci.appveyor.com/project/janisz/dcos-diagnostics/builds/21482670

Reply:
I have updated with an 'if' statement that should help clarify shifts in expected results and work on all platforms.